src/bundle/Resources/public/scss/_forms.scss

```diff
@@ -61,18 +61,12 @@ form:not(.form-inline) {
 
 .form-check {
     margin-bottom: 0;
+}
 
-    &-label {
-        display: flex;
-        align-items: center;
-        margin-bottom: 0;
-    }
-
-    &-input {
-        position: initial;
-        margin-bottom: .25rem;
-        margin-right: .5rem;
-    }
+.form-check-input {
```

Comment:
Should it be `.ez-form-check__input` ? @sunpietro

Reply:
He's overriding Bootstrap styles, so no.